src/main/java/net/kodehawa/mantarobot/commands/OwnerCmd.java

```diff
@@ -120,14 +120,15 @@ protected void call(Context ctx, String content, String[] args) {
                 }
 
                 if (context.equals("user")) {
+                    User user = MantaroBot.getInstance().getShardManager().retrieveUserById(args[2]).complete();
```

Comment:
throws when user doesn't exist, but not much of an issue in owner commands

Reply:
Hmm. Dunno if we really mind too much in owner commands, but I fixed it on most of them except the one using NewContext (at least until we migrate to the new cmd system)